compiler/include/type.h

```diff
@@ -247,8 +247,8 @@ TYPE_EXTERN PrimitiveType* dtTaskList;
 // Well-known types
 TYPE_EXTERN AggregateType* dtString;
 TYPE_EXTERN AggregateType* dtArray;
-TYPE_EXTERN AggregateType* dtReader;
-TYPE_EXTERN AggregateType* dtWriter;
+//TYPE_EXTERN AggregateType* dtReader;
+//TYPE_EXTERN AggregateType* dtWriter;
```

Comment:
Let's just remove these types instead of commenting them out.  If someone wants to undo what you're doing, they should look at the entirety of this PR anyways.